### DISCUSSION TITLE
Press gesture recognizer

### DISCUSSION THREAD

**Discussion by KSemenenko** [CommunityToolkit/Maui#622] [New Feature Discussions]
We need more gesture recognizers for maui. 
 press, move, release etc.

so like in this:
https://github.com/xamarin/Xamarin.Forms/pull/7324

**Comment by pictos** [comment]
Hey @KSemenenko, does the TouchEffect has all that you ask?
You can check the spec here 
- https://github.com/CommunityToolkit/Maui/issues/86

**Reply by KSemenenko** [reply]
It looks very good.
There's so much spec that I thought it was something else.

Thanks :)

**Comment by CliffAgius** [comment]
Closed as answered
